srcpkgs/gitea/patches/ldflags.patch

```diff
@@ -0,0 +1,11 @@
+--- Makefile	2020-07-10 12:25:58.893000000 +1000
++++ Makefile	2020-07-10 12:26:04.329000000 +1000
+@@ -84,7 +84,7 @@
+ 	endif
+ endif
+ 
+-LDFLAGS := $(LDFLAGS) -X "main.MakeVersion=$(MAKE_VERSION)" -X "main.Version=$(GITEA_VERSION)" -X "main.Tags=$(TAGS)"
++LDFLAGS := $(GOLDFLAGS) -X "main.MakeVersion=$(MAKE_VERSION)" -X "main.Version=$(GITEA_VERSION)" -X "main.Tags=$(TAGS)"
```

Comment:
Isn't it easier to unset LDFLAGS when running `make`?

I guess the build failed when you didn't have this patch?

Reply:
LDFLAGS are being used for cgo afaiu, which can indeed use them.

Reply:
Patch removed.